mods/ctf/ctf_map/ctf_map_core/nodes.lua

```diff
@@ -424,6 +424,25 @@ do
 		sounds = default.node_sound_leaves_defaults()
 	})
 
+	minetest.register_node(":ctf_map:papyrus", {
+		description = "Indestructible Papyrus",
+		drawtype = "plantlike",
+		tiles = {"default_papyrus.png"},
+		inventory_image = "default_papyrus.png",
+		wield_image = "default_papyrus.png",
+		paramtype = "light",
+		sunlight_propagates = true,
+		walkable = false,
+		selection_box = {
+			type = "fixed",
+			fixed = {-6 / 16, -0.5, -6 / 16, 6 / 16, 0.5, 6 / 16},
+		},
+		groups = {immortal = 1},
+		sounds = default.node_sound_leaves_defaults(),
+		after_dig_node = function(pos, node, metadata, digger)
+			default.dig_up(pos, node, digger)
+		end,
```

Comment:
Not needed as there are no more lines after that. 🙂
```suggestion
		end
```

Reply:
I thought it was a convention to leave a comma after the last element of a table.

Reply:
Yes it is